plugins/org.eclipse.fordiac.ide.application/src/org/eclipse/fordiac/ide/application/figures/SubAppForFbNetworkFigure.java

```diff
@@ -247,15 +249,22 @@ private void addComment() {
 		final Figure commentContainer = new Figure();
 		commentContainer.setLayoutManager(new ToolbarLayout());
 		expandedMainFigure.add(commentContainer, createCommentLayoutData(), 0);
-		// create a grid layout to get the default margin which is also used in groups
-		final GridLayout marginGetter = new GridLayout();
-		commentContainer.setBorder(new MarginBorder(marginGetter.marginHeight));
 
 		commentFigure = new InstanceCommentFigure();
-		commentFigure.setBorder(new AdvancedLineBorder(PositionConstants.SOUTH));
+		final AdvancedLineBorder commentLineSeperator = new AdvancedLineBorder(PositionConstants.SOUTH);
+		commentFigure.setBorder(commentLineSeperator);
 		commentFigure.setCursor(Cursors.SIZEALL);
 		commentContainer.add(commentFigure);
 		refreshComment();
+
+		final int lineHeight = (int) CoordinateConverter.INSTANCE.getLineHeight();
```

Comment:
This code looks similar to GroupFigure

Reply:
Yes it is similar, but that code need to be adjusted to the individual figure. As the group looks different to subapp and may also change differently I would like to keep that separated.